### PR TITLE
Feat/support partial update in update spec

### LINKED ISSUE #489: [Feature] update linear axis range by api, the same as updateDataSync()
### What problem does this feature solve?

update linear axis range using api, avoiding using updateSpec to enhance performance.

### What does the proposed API look like?

chart.updateAxisRange('axisId', { min, max });

### PR DESCRIPTION
**Pull request by xuefei1313**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close #489

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: common/changes/@visactor/vchart/feat-support-partial-update-in-updateSpec_2023-09-06-11-57.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "feat: srovide updateModelSpec api, so that users can update the configuration of a chart module individually\n\n",
```
Comment (kkxxkk2019) [inline common/changes/@visactor/vchart/feat-support-partial-update-in-updateSpec_2023-09-06-11-57.json:4]: srovide 错别字

Comment (kkxxkk2019) [reply common/changes/@visactor/vchart/feat-support-partial-update-in-updateSpec_2023-09-06-11-57.json:4]: api  文档补充上

File: packages/vchart/src/core/vchart.ts [hunk 2]
```diff
@@ -775,6 +785,109 @@ export class VChart implements IVChart {
...
 
+  /**
+   * **异步方法** spec 更新
+   * @param filter
+   * @param spec
+   * @param forceMerge
+   * @returns
+   * @sync 1.4.0
+   */
+  updateModelSpec(
```
Comment (kkxxkk2019) [inline packages/vchart/src/core/vchart.ts:796]: 这两个方法在 VChart 的接口类型定义也补充上

Comment (kkxxkk2019) [reply packages/vchart/src/core/vchart.ts:796]: 单测也写一下吧

Comment (xuefei1313) [reply packages/vchart/src/core/vchart.ts:796]: ok

**Review by kkxxkk2019 — APPROVED** [review]

**Pull request merged**